pkg/model/event.go

```diff
@@ -72,7 +72,7 @@ func (e *Event) FormatAmount(a uint64) string {
 
 // Hash Generate the event hash
 func (e *Event) Hash() string {
-	return utils.ShortHash(e.TokenSymbol, e.Owner, e.Provider)
+	return utils.ShortHash(e.TokenSymbol, e.Owner)
```

Comment:
why?

Reply:
because we could have multiple providers in the future, so what then? better keep it simple - andrea